libs/application/templates/financial-aid/src/fields/Confirmation/ApplicantConfirmation.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import { ApproveOptions, FAFieldBaseProps } from '../../lib/types'
+import { confirmation } from '../../lib/messages'
+import { hasFiles, hasSpouse } from '../../lib/utils'
+import Confirmation from './Confirmation'
+
+const ApplicantConfirmation = ({ application }: FAFieldBaseProps) => {
+  const { answers, externalData } = application
+
+  const applicantHasSpouse = hasSpouse(answers, externalData)
+  const missingIncomeFiles =
+    answers.income === ApproveOptions.Yes && !hasFiles('incomeFiles', answers)
+
+  let firstStepText
+  if (applicantHasSpouse && missingIncomeFiles) {
+    firstStepText = confirmation.nextSteps.contentBothMissingFiles
+  } else if (applicantHasSpouse) {
+    firstStepText = confirmation.nextSteps.contentSpouseMissingFiles
+  } else if (missingIncomeFiles) {
+    firstStepText = confirmation.nextSteps.contentMissingFiles
+  }
```

Comment:
should this be a switch statement?

Reply:
how do you use a switch statement with two variables? are you talking about like this:
<img width="386" alt="image" src="https://user-images.githubusercontent.com/43727721/160609901-60ee71c2-933c-4eee-a9d7-f171da705c00.png">

Reply:
yesss exactly like this ☝🏼